memgpt/utils.py

```diff
@@ -13,12 +13,14 @@
 import sqlite3
 import fitz

```

Comment:
This import made a lot of other changes important. Maybe this should get separately and not into utils too?